pkg/report/linux_test.go

```diff
@@ -511,3 +512,16 @@ func testDisassembly(t *testing.T, reporter *Reporter, linuxReporter *linux, tes
 		t.Fatalf("expected:\n%s\ngot:\n%s", output, result)
 	}
 }
+
+func TestFalseReboot(t *testing.T) {
+	_, linuxReporter := prepareLinuxReporter(t, targets.AMD64)
+
+	for _, reportCase := range []string{"691", "722", "723"} {
```

Comment:
Note that existing tests will read/check all file, including these, and will probably fail.